website/docs/language/functions/nonsensitive.mdx

```diff
@@ -73,8 +73,8 @@ due to an inappropriate call to `nonsensitive` in your module, that's a bug in
 your module and not a bug in Terraform itself.
 **Use this function sparingly and only with due care.**
 
-`nonsensitive` will return an error if you pass a value that isn't marked
-as sensitive, because such a call would be redundant and potentially confusing
+`nonsensitive` will no longer return an error if you pass a value that isn't marked
```

Comment:
I think we don't need to clarify this with `no longer` as new readers won't have the context that this used to not work.

Let's change this to:

> `nonsensitive` will make no changes to values that aren't marked as sensitive, even though such a call may be redundant and potentially confusing.

Reply:
Changed as per @liamcervante indication. Thx